docs/reference/release-notes/highlights.asciidoc

```diff
@@ -29,6 +29,42 @@ for compliance or regulatory reasons, the occasional lookback or investigation,
 or to rehydrate parts of it. Access to the data is expected to be infrequent,
 and can therefore happen with limited performance and query capabilities.
 
+[discrete]
+[[new_geo_grid_query]]
+=== New geo_grid query
+With the geo grid query, you can now natively return all the documents that overlap a specific geo tile.
+There is no need to reconstruct the geometry or the actual boundaries of the spatial cluster as Elasticsearch
+can do this for you which saves you time and reduces complexity.
+This is especially useful when geometries are spread across tiles like on a soccer ball football.
+While hexagon tiles line the sphere each tile is not exactly straight forward to calculate its boundary.
```

Comment:
```suggestion
There is no need to reconstruct the geometry or the actual boundaries of the spatial cluster as Elasticsearch
can do this for you, which saves you time and reduces complexity.
This is especially useful when geometries are spread across tiles like on a soccer ball or football.
While hexagon tiles line the sphere, calculating the boundary of each tile is not straightforward.
```

Reply:
The text came from PM from a planned blog, but I think your improvements are great, so I just added them. We'll see if PM want to improve the blog later.